build-system/server/app-utils.js

```diff
@@ -127,51 +127,51 @@ const replaceUrls = (mode, file, hostName, inabox, storyV1) => {
     }
   } else if (mode == 'compiled') {
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\.js/g,
-      hostName + '/dist/v0.js'
+      /https:\/\/cdn\.ampproject\.org\/v0\.(m?)js/g,
+      hostName + '/dist/v0.$1js'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/shadow-v0\.js/g,
-      hostName + '/dist/shadow-v0.js'
+      /https:\/\/cdn\.ampproject\.org\/shadow-v0\.(m?)js/g,
+      hostName + '/dist/shadow-v0.$1js'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/amp4ads-v0\.js/g,
-      hostName + '/dist/amp4ads-v0.js'
+      /https:\/\/cdn\.ampproject\.org\/amp4ads-v0\.(m?)js/g,
+      hostName + '/dist/amp4ads-v0.$1js'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/video-iframe-integration-v0\.js/g,
-      hostName + '/dist/video-iframe-integration-v0.js'
+      /https:\/\/cdn\.ampproject\.org\/video-iframe-integration-v0\.(m?)js/g,
+      hostName + '/dist/video-iframe-integration-v0.$1js'
     );
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\/(.+?).js/g,
-      hostName + '/dist/v0/$1.js'
+      /https:\/\/cdn\.ampproject\.org\/v0\/(.+?).(m?)js/g,
+      hostName + '/dist/v0/$1.$2js'
     );
     file = file.replace(
-      /\/dist\/v0\/examples\/(.*)\.max.js/g,
-      '/dist/v0/examples/$1.js'
+      /\/dist\/v0\/examples\/(.*)\.max.(m?)js/g,
+      '/dist/v0/examples/$1.$2js'
     );
     file = file.replace(
       /\/dist.3p\/current\/(.*)\.max.html/g,
       hostName + '/dist.3p/current-min/$1.html'
     );
 
     if (inabox) {
-      file = file.replace(/\/dist\/v0\.js/g, '/dist/amp4ads-v0.js');
+      file = file.replace(/\/dist\/v0\.(m?)js/g, '/dist/amp4ads-v0.$1js');
     }
   } else if (isRtvMode(mode)) {
     hostName = `https://cdn.ampproject.org/rtv/${mode}/`;
     file = file.replace(/https:\/\/cdn\.ampproject\.org\//g, hostName);
 
     if (inabox) {
       file = file.replace(
-        /https:\/\/cdn\.ampproject\.org\/rtv\/\d{15}\/v0\.js/g,
-        hostName + 'amp4ads-v0.js'
+        /https:\/\/cdn\.ampproject\.org\/rtv\/\d{15}\/v0\.(m?)js/g,
+        hostName + 'amp4ads-v0.$1js'
       );
     }
   } else if (inabox) {
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\.js/g,
-      'https://cdn.ampproject.org/amp4ads-v0.js'
+      /https:\/\/cdn\.ampproject\.org\/v0\.(m?)js/g,
```

Comment:
Nit: Here and below maybe `(m?js)` and the resulting replacement `.$1` would be slightly more readable

Reply:
done

Reply:
It looks like maybe GitHub doesn't have the most up-to-date version of this file/branch?

Reply:
sorry. pushed it out again. was having trouble because i committed your changes through the webui and forgot my shortcuts were all "--force-with-lease" and things weren't getting pushed

Reply:
I'm still not seeing changes to this file :thinking: